gen-dirs.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+set -e
+cd templates
+FILES=$(find -follow)
```

Comment:
Hm, I dislike this, tbh.  Especially that we filter everything through the build script, while simple copy would often be enough.

Maybe we could separate the real templates into ./templates, and normal files under ./files?

Other approach could be using some "manifest file", I implemented something like this in https://github.com/devexp-db/cont-postgresql/blob/master/cl-manifest
That file declars what should be copied, what should be "instantiated" (and by what tool) and everything explicitly (not copying some accidentally occurred files). This should stay executed by postgresql "in parallel" if implemented.